docs/Introduction.AndroidDevEnv.md

```diff
@@ -15,26 +15,26 @@ Note that running automated UI tests is _not the same_ as developing Android app
 
```

Comment:
Since RN 66 may not be the golden standard out there just yet, we should, at least for a while, explain that Java11 is required in order to support RN66+Android12, and that otherwise, J8 should be used, etc.